mesh/twitter_profile_agent.py

```diff
@@ -0,0 +1,496 @@
+                "author_address": "0x7d9d1821d15B9e0b8Ab98A058361233E255E405D",
+                "description": "This agent fetches a Twitter user's profile information and recent tweets. It's useful for getting project updates or tracking key opinion leaders (KOLs) in the space.",
+                "inputs": [
+                    {
+                        "name": "query",
+                        "description": "Twitter username (with or without @) or user ID to fetch profile and tweets for",
+                        "type": "str",
+                        "required": False,
+                    },
+                    {
+                        "name": "raw_data_only",
+                        "description": "If true, return only raw data without natural language response",
+                        "type": "bool",
+                        "required": False,
+                        "default": False,
+                    },
+                ],
+                "outputs": [
+                    {
+                        "name": "response",
+                        "description": "Natural language summary of the user's profile and recent tweets",
+                        "type": "str",
+                    },
+                    {
+                        "name": "data",
+                        "description": "Structured data containing user profile and recent tweets",
+                        "type": "dict",
+                    },
+                ],
+                "external_apis": ["Twitter API"],
+                "tags": ["Social", "Twitter"],
+                "recommended": True,
+                "image_url": "https://raw.githubusercontent.com/heurist-network/heurist-agent-framework/refs/heads/main/mesh/images/Twitter.png",
+                "examples": [
+                    "Get recent tweets from @username",
+                    "Show me the latest updates from username",
+                    "What has username been tweeting lately?",
+                    "Get the recent tweets from username",
+                    "Check the latest tweets from user_id:123456789",
+                ],
+            }
+        )
+
+    def get_system_prompt(self) -> str:
+        return """
+        You are a specialized Twitter analyst that helps users get information about Twitter profiles and their recent tweets.
+
+        When analyzing a Twitter profile:
+        1. Provide a brief overview of the account (name, bio, follower count)
+        2. Summarize their recent tweet activity (frequency, topics)
+        3. Highlight notable tweets with high engagement
+
+        Keep your analysis factual and concise. Focus on the data provided without making assumptions.
+        """
+
+    def get_tool_schemas(self) -> List[Dict]:
+        return [
+            {
+                "type": "function",
+                "function": {
+                    "name": "get_user_tweets",
+                    "description": "Fetch recent tweets from a Twitter user",
```

Comment:
add more details: "Fetch recent tweets from a Twitter user. Use this tool to get project updates or tracking recent posts by key opinion leaders (KOLs)"